Applications/Data/README.md

```diff
@@ -0,0 +1,63 @@
+# Faraday Data Server Proxy Application Documentation
```

Comment:
NOTE: This program is in active development. It is possible that bugs are present and features not yet added. Please report any bugs found or features that you feel should be included in the next release.